policyengine_us/variables/gov/local/ca/la/general_relief/eligibility/la_general_relief_immigration_status_eligible.py

```diff
@@ -0,0 +1,21 @@
+from policyengine_us.model_api import *
+
+
+class la_general_relief_immigration_status_eligible(Variable):
+    value_type = bool
+    entity = SPMUnit
+    definition_period = YEAR
+    label = "Eligible for the Los Angeles County General Relief based on the immigration status requirements"
+    # Person has to be a resident of LA County
+    defined_for = "in_la"
+
+    def formula(spm_unit, period, parameters):
+        person = spm_unit.members
+        # person = spm_unit.members
+        # Undocuemnted as well as DACA classified filers are ineligible for the GR
+        istatus = person("immigration_status", period)
+        daca_tps = istatus == istatus.possible_values.DACA_TPS
+        undocumented = istatus == istatus.possible_values.UNDOCUMENTED
+        # Assuming that the filers immigartion status is recorded
+        ineligible_status = daca_tps | undocumented
+        return ~spm_unit.any(ineligible_status)
```

Comment:
Households are ineligible if anyone is daca/tps/undocumented? Even if some are citizens?

Reply:
Unless they're highly complicated, let's match the rules; SBN can pass subsets of inputs as needed

Reply:
I am reading the [below](http://epolicy.dpss.lacounty.gov/epolicy/epolicy/server/general/projects_responsive/ePolicyMaster/index.htm?&area=general&type=responsivehelp&ctxid=&project=ePolicyMaster#t=mergedProjects%2FGR%2FGR%2F42-404_Immigrant_Eligibility_Chart%2F42-404_Immigrant_Eligibility_Chart.htm) as "everybody in the SpmUnit must be a a United States (U.S.) citizen or a qualified noncitizen". Let me know if I am misinterpreting

![image](https://github.com/PolicyEngine/policyengine-us/assets/110687043/c49a84c7-098b-41db-91ba-fa1b7d2b3078)